### PR TITLE
fix: next example now works

### PR DESCRIPTION
**Pull request by multivoltage**
### Fixed
For my previus PR I noticed that nextjs did not work since it use a different react version.
I fixed using same version.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
```
Comment (devrnt) [inline pnpm-lock.yaml:6]: Any reason why this was added?

**Review by devrnt — COMMENTED** [review]
Thanks man! I've 1 question, see review

File: pnpm-lock.yaml (hunk 1 above)
Comment (multivoltage) [reply pnpm-lock.yaml:6]: Is my second time with pnpm. 
Reading here https://github.com/pnpm/pnpm/releases/tag/v7.33.0 means that my installation of pnpm added these flag. I need to remove?

Comment (devrnt) [reply pnpm-lock.yaml:6]: I see, same happened for [version 8.x.x](https://github.com/pnpm/pnpm/releases/tag/v8.6.0)

**Comment by devrnt** [comment]
I ran this locally and seems like this is still not working

**Comment by multivoltage** [comment]
What task did you run?

**Comment by multivoltage** [comment]
@devrnt 
I installed  pnpm@7.26.3 and reverted these 4 lines.
Then I Run "build" "build-playground" and "test" without error.

**Comment by devrnt** [comment]
I'm running the `dev:examples` script. Seems like there are 2 versions of React in the `nextjs` example

**Comment by multivoltage** [comment]
I don't know.
Before my commit there was 2 version of react between next and gatasby. Now There is only 18.2.0.
I delete and clone again my branch.
Run:
pnpm run install, pnpm run dev, pnpm run dev-examples, pnpm run build.
All these task finish ok in my local ;(

File: apps/examples/nextjs/package.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "12.1.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "next": "12.3.4",
```
Comment (devrnt) [inline apps/examples/nextjs/package.json:10]: While we are at it can we bump this to the latest `13.x` version?

**Review by devrnt — CHANGES_REQUESTED** [review]
My bad, had to do a clean re-install of my dependencies. Seems to work fine. Just one more thing

File: apps/examples/nextjs/package.json (hunk 2 above)
Comment (devrnt) [reply apps/examples/nextjs/package.json:10]: Make sure to run `pnpm add --filter nextjs-example react@latest react-dom@latest next@latest` in folder root :)

Comment (devrnt) [reply apps/examples/nextjs/package.json:10]: It's fine if the lock-file changes again. Don't mind the `settings` block being added, that's expected as you mentioned

Comment (devrnt) [reply apps/examples/nextjs/package.json]: I see, thanks. About the `settings` entry in the lockfile. It was introduced in [8.6.0](https://github.com/pnpm/pnpm/releases/tag/v8.6.0), but it was rolled back in (https://github.com/pnpm/pnpm/releases/tag/v8.6.2)[8.6.2], so nevermind my comment

**Review by devrnt — APPROVED** [review]

**Comment by devrnt** [comment]
Big thanks!

**Pull request merged**